tests/test_scripts.py

```diff
@@ -2777,6 +2776,49 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting_load_gzipped_bigcount():
+    infile = utils.get_temp_filename('test_ct')
+    ct = khmer.new_counting_hash(10, 1e5, 4)
+    ct.set_use_bigcount(True)
+    for i in range(500):
+        ct.count('ATATATATAT')
+    ct.save(infile)
+    outfile = infile + '.gz'
+    data = open(infile, 'rb').read()
+    f_out = gzip.open(outfile, 'wb')
+    f_out.write(data)
+    f_out.close()
+    newct = khmer.new_counting_hash(10, 1e5, 4)
+    newct.load(outfile)
+    count = newct.get('ATATATATAT')
+    assert count == 500
+
+
+def test_abundance_distribution_gzipped_bigcount():
```

Comment:
I think this test should be changed to look only at bigcount behavior test in counting_hash.abundance_distribution, which is untested elsewhere; your other test function does a fine job of testing bigcount load in gzip files.

So, remove all of the hashtable save/load stuff, and just make it about bigcount testing.

Also, this function should be moved to test_counting_hash.py, because it doesn't test a script.